docs/monitoring-metrics.md

```diff
@@ -24,7 +24,21 @@ or APIs ([here](https://www.elastic.co/guide/en/kibana/current/dashboard-import-
 |------------------------------------------------|----------------------------------|
 | <img alt="Jenkins Health Dashboard with Elastic Kibana" width="300px" src="https://raw.githubusercontent.com/jenkinsci/opentelemetry-plugin/master/docs/images/kibana_jenkins_overview_dashboard.png" /> | <img alt="Jenkins Agent Provisioning Health Dashboard with Elastic Kibana" width="300px" src="https://raw.githubusercontent.com/jenkinsci/opentelemetry-plugin/master/docs/images/kibana_jenkins_provisioning_dashboard.png" /> |
 
-## Jenkins Health Metrics
+## Build Duration
+
+* Name: `ci.pipeline.run.duration`
+* Type: Histogram with buckets: `1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192` (buckets subject to change)
+* Unit: `s`
+* Attributes:
+  * `ci.pipeline.id`: The full name of the Jenkins job if complying with the allow and deny lists specified through 
+    configuration parameters documented below, otherwise `#other#` to limit the cardinality of the metric. 
+  Example: `my-team/my-app/main`. See `hudson.model.AbstractItem#getFullName()`.
+  * `ci.pipeline.result`: `SUCCESS`, `UNSTABLE`, `FAILUIRE`, `NOT_BUILT`, `ABORTED`. See `hudson.model.Run#getResult()`.
+* Configuration parameters to control the cardinality of the `ci.pipeline.id` attribute:
+  * `otel.instrumentation.jenkins.run.metric.duration.allow_list`: Java regex. Example `jenkins_folder_a/.*|jenkins_folder_b/.*`
+  * `otel.instrumentation.jenkins.run.metric.duration.deny_list`: Java regex. Example `.*test.*`
```

Comment:
Maybe specify the defaults for these properties?

Reply:
Please review updated docs. I tagged you on the improved sections that describe this behavior.

Reply:
The updated docs look good to me.